repo/tuf_on_ci/_repository.py

```diff
@@ -540,13 +570,24 @@ def update_targets(self, rolename: str) -> bool:
 
         return False
 
-    def is_verified(self, rolename: str) -> bool:
+    def is_valid(self, rolename: str) -> bool:
```

Comment:
may have to rethink the name here:
* there is already a `_validate_role()` that is used during a signing event (so takes into account the signing event starting point)
* this one is more about "does this role need to be signed" and is currently only used during online sign